controllers/profile_controller_test.go

```diff
@@ -151,7 +154,7 @@ var _ = Describe("ProfileController", func() {
 				subscriptionName := "git-resource-already-exists-error"
 				profileURL := nginxProfileURL
 
-				gitRefName := fmt.Sprintf("%s-%s-%s", subscriptionName, "nginx-profile", "main")
+				gitRefName := fmt.Sprintf("%s-%s-%s", subscriptionName, "nginx-profile", "support-helm-urls")
```

Comment:
TODO: We need to change / deal with this. This is currently pinning to a repository here: https://github.com/weaveworks/nginx-profile/pull/6 which need to change the chart value to HelmChartLocal.